runners/s3-benchrunner-rust/src/main.rs

```diff
@@ -1,15 +1,44 @@
-use clap::{Parser, ValueEnum};
-use std::process::exit;
+use clap::{Parser, Subcommand, ValueEnum};
+use std::fs::File;
 use std::time::Instant;
+use std::{path::Path, process::exit};
 use tracing::{info_span, Instrument};
 
+use opentelemetry_proto::tonic::{
+    collector::trace::v1::{trace_service_client::TraceServiceClient, ExportTraceServiceRequest},
+    trace::v1::TracesData,
+};
+use tonic::transport::Channel;
+
 use s3_benchrunner_rust::{
     bytes_to_gigabits, prepare_run, telemetry, BenchmarkConfig, Result, RunBenchmark,
     SkipBenchmarkError, TransferManagerRunner,
 };
+
+#[derive(Parser, Debug)]
+struct SimpleCli {
+    #[command(flatten)]
+    run_args: RunArgs,
+}
+
 #[derive(Parser, Debug)]
-#[command()]
-struct Args {
+struct ExtendedCli {
+    #[command(subcommand)]
+    command: Command,
+    #[command(flatten)]
+    run_args: Option<RunArgs>,
+}
+
+#[derive(Subcommand, Debug)]
+enum Command {
+    RunBenchmark(RunArgs),
```

Comment:
trivial/debatable: if we stick with this, I'd prefer a shorter name, since this is what we're doing 99% of the time
```suggestion
    Run(RunArgs),
```